resources/prometheus-ro-rootfs.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Run prometheus-ro-rootfs service periodically
+After=prometheus-tmpfs-dir.service
+Requires=prometheus-tmpfs-dir.service
+
+[Timer]
+OnBootSec=60
+OnUnitInactiveSec=10
```

Comment:
We scrape every 30s, so I don't think we'll benefit from this kind of precision. Set it to `30s` as well?